private/buf/bufgen/bufgen.go

```diff
@@ -169,6 +169,7 @@ type ManagedConfig struct {
 	JavaPackagePrefix     string
 	OptimizeFor           *descriptorpb.FileOptions_OptimizeMode
 	GoPackagePrefixConfig *GoPackagePrefixConfig
+	Overrides             map[string]map[string]string
```

Comment:
I would call this `override` personally. We say `except`, not `exceptions`, in our config files. Thoughts? cc @amckinney

Reply:
Agreed, `override` is also symmetrical with `managed.go_package_prefix.override`.